packages/backend-plugin-api/src/services/definitions/coreServices.ts

```diff
@@ -78,6 +78,13 @@ export namespace coreServices {
     import('./DiscoveryService').DiscoveryService
   >({ id: 'core.discovery' });
 
+  /**
+   * The service reference for the plugin scoped {@link HealthService}.
+   */
+  export const health = createServiceRef<
+    import('./HealthService').HealthService
+  >({ id: 'core.health', scope: 'root' });
```

Comment:
```suggestion
   * The service reference for the root scoped {@link RootHealthService}.
   */
  export const rootHealth = createServiceRef<
    import('./RootHealthService').RootHealthService
  >({ id: 'core.rootHealth', scope: 'root' });
```

Every single other root scoped thing is named like this - we should probably follow suit here. Make sure to change to that naming pattern everywhere else too.

I'd also be interested in what the future might be in terms of per-plugin liveness/health. That's sort of where the most interesting developments might be in the future. This root scoped one could at most do simple lifecycle matching like it does today, or _possibly_ offer some form of rollup of the states of all installed plugins (not even sure that's a good idea though). I mean, how often do we really desire to look at the liveness of effectively _a machine_ that hosts many plugins? What does that even mean?